api/app/Providers/GraphQLServiceProvider.php

```diff
@@ -3,11 +3,641 @@
 namespace App\Providers;
 
 use App\GraphQL\Operators\PostgreSQLOperator;
+use GraphQL\Type\Definition\Description;
+use GraphQL\Type\Definition\EnumType;
 use Illuminate\Support\ServiceProvider;
+use Nuwave\Lighthouse\Schema\TypeRegistry;
 use Nuwave\Lighthouse\WhereConditions\Operator;
 
+enum Language: string
```

Comment:
Having all the enums declared directly in the App\Providers namespace and in this service provider file doesn't seem quite clean to me. (This page)[https://www.itsolutionstuff.com/post/how-to-use-enum-in-laravelexample.html] suggests app/Enums which seems reasonable to me.  Alternatively, you could leave these definitions in the regular ApiEnums file.  Maybe worth a quick discussion in scrum today.

Reply:
It looks like [the BenSampo library](https://github.com/BenSampo/laravel-enum#basic-usage) that covered enums before they came natively to PHP also uses App\Enums.

Reply:
These have been moved to a new app\Enums folder with [849ee42](https://github.com/GCTC-NTGC/gc-digital-talent/pull/7923/commits/849ee4285d5994748aba12e705bf5d7c7f3b2210).